Dockerfile

```diff
@@ -142,19 +142,17 @@ ENV \
 	OSDK_BIN=/usr/local/osdk/bin
 
 # Install the certsuite binary
-COPY --from=build ${CERTSUITE_DIR} ${CERTSUITE_DIR}
```

Comment:
So `CERTSUITE_DIR` is the actual DIR where the `CERTSUITE_DIR/certsuite` is the binary.  I think you just need: 

`COPY --from=build ${CERTSUITE_DIR}/certsuite /usr/local/bin/certsuite`

Reply:
ahh I was reading it as the binary name, but that makes sense based on the `tree` output.

Reply:
Although, now that I think about it, I think you might hit a problem where you also need the `rhcos_version_map` file (tests/platform/operatingsystem/files/rhcos_version_map) if you aren't copying the entire directory for that test to work.

Reply:
@sebrandon1 The code shows that `go:embed` is being used to include the file at binary build time, is that not working correctly?